esmvaltool/recipes/examples/recipe_check_obs.yml

```
@@ -548,16 +548,16 @@ diagnostics:
       thetao:
         mip: Omon
       no3:
-        mip: Oyr
+        mip: Omon
       o2:
-        mip: Oyr
+        mip: Omon
       po4:
-        mip: Oyr
+        mip: Omon
       si:
-        mip: Oyr
+        mip: Omon
     additional_datasets:
-      - {dataset: WOA, project: OBS, tier: 2,
-         type: clim, version: 2013v2, start_year: 2000, end_year: 2000}
+      - {dataset: WOA, project: OBS6, tier: 2,
+         type: clim, version: 2018, start_year: 2000, end_year: 2000}
```

```suggestion
dataset: WOA, project: OBS6, tier: 2, mip: Omon
```
just an idea to avoid repeating `mip: Omon` for every variable and improve readability

---

that's ok ...

---

I added the suggested change in recipe_check_obs.yml, see fbefb8e